pkg/dexidp/provider.go

```diff
@@ -31,9 +31,11 @@ func (p *dexProvider) Metadata(ctx context.Context, req provider.MetadataRequest
 // Schema defines the provider-level schema for configuration data.
 func (p *dexProvider) Schema(ctx context.Context, req provider.SchemaRequest, resp *provider.SchemaResponse) {
 	resp.Schema = schema.Schema{
+		Description: "Provision Dex resources via the Dex gRPC API.",
 		Attributes: map[string]schema.Attribute{
 			"host": schema.StringAttribute{
```

Comment:
since the description mentions host and port, I am thinking about renaming this attribute to address.

Reply:
as dex gRPC api also names this host I think we should keep that naming.